dags/als-kp-dataset-dag.py

```diff
@@ -325,7 +381,7 @@ def create_datasets(
         ).get(synapse_client=synapse_client)
```

Comment:
`synapse_client=synapse_client` can be removed for consistency throughout the script